packages/client/tests/functional/metrics/disabled/tests.ts

```diff
@@ -6,14 +6,14 @@ import type { PrismaClient } from './node_modules/@prisma/client'
 
 declare let prisma: PrismaClient
 
-testMatrix.setupTestSuite(() => {
-  test('attempt to use $metrics a compile-time error', () => {
+testMatrix.setupTestSuite((_0, _1, _2, cliMeta) => {
+  skipTestIf(cliMeta.previewFeatures.includes('metrics'))('attempt to use $metrics a compile-time error', () => {
```

Comment:
Because now we force the metrics flag, this test needs to handle it. It tests what should happen when metrics preview feature is not enabled.

Reply:
Do you think it would be common enough usecase? If it is, would be better to add it as the rest of `optOut` options with a forced `reason`?

Reply:
Yes, I think we should clean this up